src/postgresql/pgopt.tcl

```diff
@@ -238,20 +238,12 @@ set Name $Parent.f1.e8
    ttk::entry $Name -width 30 -textvariable pg_dbase
    grid $Prompt -column 0 -row 8 -sticky e
    grid $Name -column 1 -row 8 -sticky ew
-if { $option eq "all" || $option eq "build" } {
-set Name $Parent.f1.e8a
-   set Prompt $Parent.f1.p8a
-   ttk::label $Prompt -text "PostgreSQL Tablespace :"
-   ttk::entry $Name -width 30 -textvariable pg_tspace
```

Comment:
It seems this PR accidentally undoes the table space changes for PostgreSQL in #95.

That causes errors when trying to run TPC-C against PostgreSQL:
```
Error: can't read "pg_tspace": no such variable
```